packages/ckeditor5-editor-multi-root/src/multirooteditor.ts

```diff
@@ -231,11 +229,21 @@ export default class MultiRootEditor extends DataApiMixin( Editor ) {
 			}
 		} );
 
-		this.listenTo<ModelIsEditableEvent>( this.model, 'isEditable', ( evt, [ selectable ] ) => {
-			const isEditable = this._isEditable( selectable );
-			evt.return = isEditable;
+		// Overwrite `Model#canEditAt()` decorated method.
+		// Check if the provided selection is inside a read-only root. If so, return `false`.
+		this.listenTo<ModelCanEditAtEvent>( this.model, 'canEditAt', ( evt, [ selection ] ) => {
+			// Skip empty selections.
+			if ( !selection.getFirstPosition() ) {
+				return;
+			}
+
+			const rootName = selection.getFirstPosition()!.root.rootName!;
```

Comment:
What if there are multiple roots coming from move to graveyard? Should we check all roots?